src/app/campaign.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface Campaign {
```

Comment:
Interfaces are used only at TypeScript compile time. It seems like Angular encourages classes so it's probably best to use these for all models for consistency https://stackoverflow.com/questions/37652801/when-to-use-interface-and-model-in-typescript-angular2

I think you will also find when writing unit tests for the new parts that you can't make the objects you need without implementing or replacing the interface